Update README.md (#25)
fivetran/dbt_apple_search_ads: README.md

| [apple_search_ads__keyword_report](https://fivetran.github.io/dbt_apple_search_ads/#!/model/model.apple_search_ads.apple_search_ads__keyword_report)            | Each record in this table represents the daily performance at the ad group level for keywords. |
| [apple_search_ads__search_term_report](https://fivetran.github.io/dbt_apple_search_ads/#!/model/model.apple_search_ads.apple_search_ads__keyword_report)            | Each record in this table represents the daily performance at the ad group level for search term report. |

### Materialized Models

> **26** models will be materialized in each run when all components of this data model are enabled. This count includes all staging, intermediate, and final models materialized as either `view` or `table`.

Reviewer: ```suggestion
Each transformation job run materializes **26** models if you enable all components of this data model. This count includes all staging, intermediate, and final models materialized as either `view` or `table`.
```

Reviewer: @fivetran-joemarkiewicz Should we be more specific about what is not included in that count (e.g., test models)? Or is that not relevant in this context?

Author: I don't imagine that's entirely necessary and may cause more confusion. Especially since tests are not materialized as tables of views.

Reviewer: @fivetran-joemarkiewicz Makes sense. Let me know if you're OK with the suggested changes. If yes, we can use that format as a type of boilerplate text for all data model docs.

Author: This looks good to me! It's direct and clearly outlines the maximum number of materialized models within the data model.

For data models where there are multiple upstream sources we would use an abridged version of this text to reference a table below that includes the upstream data model and the number of materialized models.

Reviewer: Sounds good!
